Tests/SwiftFrameTests/Utility/TestHelpers.swift

```diff
@@ -4,7 +4,13 @@ import Foundation
 // MARK: - CGSize
 
 extension CGSize {
+
     static let square100Pixels: CGSize = CGSize(width: 100, height: 100)
+
+    static func makeStandardSize(with gapWidth: Int, numberOfGaps: Int) -> CGSize {
```

Comment:
"standard" is a bit unintuitive here. Maybe better to encode the `100Pixels` into it somehow like you did with the property above.